nomo-webon-kit/src/dart_interface.ts

```diff
@@ -4,6 +4,33 @@ declare global {
   }
 }
 
+let focusedElement: HTMLInputElement | null = null;
+
+const attachFocusHandlers = () => {
```

Comment:
a few checks still need to be added.
First, the webon-kit should check if window is undefined before using window: if (typeof window !== "undefined") {
Secondly, it should check whether the code is running in a Hub.